integration-tests/flipper/lib.rs

```diff
@@ -110,5 +113,29 @@ pub mod flipper {
 
             Ok(())
         }
+
+        #[ink_e2e::test(backend = "network")]
```

Comment:
this test will take over 60s to complete. Should we gate it so it does not run in CI for now?

Reply:
Those 60s includes the network spawning?

Reply:
yes that is the biggest chunk.